src/Core/SonarLintNotifications/NotificationSetting.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SonarLint.VisualStudio.Core.SonarLintNotifications
+{
+    public interface INotificationSetting
```

Comment:
Wouldn't it make sense to have a file that only contains "do not show" notifications?
What is the benefit of storing notifications with DoNotShow=false?
Also, double-negative is hard to understand.

Reply:
I think there could be other info that we could store. I think it's better to save info than not saving. And for the name I couldn't find a better name. Maybe "DoNotShowSelected" to avoid double negative?